CoreScriptsRoot/Modules/VR/VirtualKeyboard.lua

```diff
@@ -0,0 +1,678 @@
+-- VirtualKeyboard.lua --
+-- Written by Kip Turner, copyright ROBLOX 2016 --
+
+
+local CoreGui = game:GetService('CoreGui')
+local RunService = game:GetService('RunService')
+local UserInputService = game:GetService('UserInputService')
+local GuiService = game:GetService('GuiService')
+local HttpService = game:GetService('HttpService')
+local ContextActionService = game:GetService('ContextActionService')
+local PlayersService = game:GetService('Players')
+
+local RobloxGui = CoreGui:WaitForChild("RobloxGui")
+local Util = require(RobloxGui.Modules.Settings.Utility)
+
+local vrKeyboardSuccess, vrKeyboardFlagValue = pcall(function() return settings():GetFFlag("UseVRKeyboardInLua") end)
+local useVRKeyboard = (vrKeyboardSuccess and vrKeyboardFlagValue == true)
```

Comment:
we could probably just exit the script here by returning an empty table right? Or is the intention that this could be used by more than just vr?